repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -528,3 +528,13 @@ export const addFlatFareProductIfNotPresent = (): void => {
         }
     });
 };
+
+export const checkAndRepickService = (): void => {
+    cy.get('body').then(($body) => {
+        if ($body.text().includes('There is a problem')) {
```

Comment:
Is this too vague, will it catch other potential issues we'd immediately want to fail the test on?

Would be good to check against the error detail "As your service only operates in a single direction..."

Reply:
yeah, I'll make these changes